scripts/style_check.sh

```diff
@@ -235,6 +235,24 @@ sev[47]=MAYOR; cat[47]='Extra space before )'
 sev[48]=NONE;  cat[48]="LAST"
 
 
+
+# -----------------------------------------------------------------------------
+#
+# Exceptions per library
+#
+# o Rule 6 for mongoBackend
```

Comment:
The 6 is the index in the internal vectors in style_check.sh. Only.
I picked "Rule 6" instead of writing the entire text about "int64/int32 etc".
If you prefer, we can use the text instead. Longer but clearer